testing/conftest.py

```diff
@@ -33,12 +33,17 @@ def pytest_sessionstart():
         install_named_dataset('sct_testing_data', dest_folder=sct_test_path())
 
 
-@pytest.fixture
-def run_in_sct_testing_data_dir():
-    """Temporarily change the working directory to 'sct_testing_data'. This replicates the behavior of the old
-    `sct_testing`, and is needed to prevent tests from cluttering the working directory with output files."""
+@pytest.fixture(autouse=True)
```

Comment:
Ah! I was wondering why the commit description said "replace", even though it looked more like "delete", but I see that this says `autouse=True`. I didn't know this was a possibility. Oh, and I see this is also how we run the integrity check! Good to know all around.